data/en/invoke.json

```diff
@@ -25,6 +25,24 @@
 			"description":"Invokes the size method on a new HashMap object, which should return 0",
 			"code":"invoke(createObject(\"java\", \"java.util.HashMap\"), \"size\")",
 			"result":"0"
+		},
+		{
+			"title":"Invoke a method on a component",
+			"description":"Invokes the method named 'test' on the component Test.cfc with one parameter",
+			"code":"obj = createObject(\"component\", \"Test\");\rinvoke(obj, \"test\", {parameter=\"Test Data\"});",
```

Comment:
You normally wouldn't have `\r` alone. It would either be `\r\n`, or preferably just `\n`. Same applies below.